src/Analyzers/CSharp/Tests/QualifyMemberAccess/QualifyMemberAccessTests.cs

```diff
@@ -2017,8 +2017,7 @@ public void Bar()
 CodeStyleOptions2.QualifyFieldAccess);
         }
 
-        [Fact]
-        [WorkItem("https://github.com/dotnet/roslyn/issues/22776")]
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/22776")]
```

Comment:
>[Fact, WorkItem [](http://example.com/codeflow?start=8&length=15)

Do we normally do this when there are multiple workitems?

Reply:
we do either.  the number of cases where we have multiple workitems is vanishly small (less than 1:1000 tests).